pom.xml

```diff
@@ -366,6 +366,24 @@
             <version>${powermock.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
+            <version>2.3.1</version>
+            <scope>test</scope>
```

Comment:
test or provided? I think it's required on runtime as well

Reply:
It's required by Hibernate <=5.3 itself in the runtime. Unfortunately, these versions don't define these dependencies. It's fixed in 5.4+ https://in.relation.to/2018/09/13/using-hibernate-orm-with-jdk11/

I'm not sure if we should fix it or rather leave it up to the developers

Reply:
I think we should leave it to the developers. Thanks for the clarification